include/ripples/graph.h

```diff
@@ -591,6 +591,12 @@ class Graph {
 
   template <typename FStream>
   void load_binary(FStream &FS) {
+    #ifdef ENABLE_METALL
+    // Static assert 0
+    throw 0 && "Not implemented yet, don't use with Metall";
```

Comment:
Runtime error, so it can be compiled, but not run; otherwise, we would need to disable compilation of many other programs outside of imm.cc utilizing load_binary.